### PR TITLE
feat: added tooltip to user icon on history dataset commit

### LINKED ISSUE #392: Username icon should show username as aa tooltip
image

### PR DESCRIPTION
**Pull request by boandriy**
added tooltip to user icon on history dataset commit

closes #392

### CODE REVIEW

File: src/chrome/UsernameWithIcon.tsx [hunk 1]
```diff
@@ -6,21 +6,23 @@ interface UsernameWithIconProps {
...
   username,
   className,
   iconWidth = 18,
-  iconOnly = false
+  iconOnly = false,
+  tooltip = false
 }) => (
    <div className={classNames('flex items-center tracking-wider', className)}>
-     <div className='rounded-xl inline-block bg-cover flex-shrink-0' style={{
+     <div className='rounded-xl inline-block bg-cover flex-shrink-0' title={tooltip ? username : undefined} style={{
```
Comment (ramfox) [inline src/chrome/UsernameWithIcon.tsx]: `title={tooltip ? username : undefined}` => `title={tooltip && username}`

don't use `undefined` as an expression of the ternary operator

**Review by ramfox — APPROVED** [review]
One fix that needs to be changed before it can merge. Once that is fixed it LGTM!

File: src/chrome/UsernameWithIcon.tsx (hunk 1 above)
Comment (boandriy) [reply src/chrome/UsernameWithIcon.tsx]: @ramfox , That was my first approach, but had an error with TS:  `Type 'string | false' is not assignable to type 'string | undefined'`

Comment (ramfox) [reply src/chrome/UsernameWithIcon.tsx]: then `tooltip ? usename : ''` should work instead

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#title_attribute_inheritance

having the title attribute as the empty string will not show a title

**Pull request merged**